src/template/gantt.tmpl.html

```diff
@@ -130,7 +130,7 @@
     <!-- Task template -->
     <script type="text/ng-template" id="template/ganttTask.tmpl.html">
         <div class="gantt-task" ng-class="task.model.classes">
-            <gantt-task-background></gantt-task-background>
+            <gantt-task-background ng-class="{'gantt-task-overlaps':task.model.overlaps}"></gantt-task-background>
```

Comment:
Please keep the template file unchanged, and handle CSS class changes using JQLite from javascript plugin code